web/Cargo.toml

```diff
@@ -36,6 +35,7 @@ ruffle_render_canvas = { path = "../render/canvas", optional = true }
 ruffle_web_common = { path = "common" }
 ruffle_render_webgl = { path = "../render/webgl", optional = true }
 ruffle_render_wgpu = { path = "../render/wgpu", optional = true }
+ruffle_video_software = { path = "../video/software" }
```

Comment:
nit: Why is it optional on desktop, but non-optional on web?

Reply:
I don't know :)
It's existing behaviour. Desktop had features to toggle it on, Web had it forcefully enabled. I just copied that.